CoreDataAccessView/release/modules/ext/docs/CoreDataAccessView/src/au/gov/asd/tac/constellation/views/dataaccess/extract-types-from-text.md

```diff
@@ -16,3 +20,22 @@ New nodes are added with the following attributes:
 ## Source Node
 
 -   Seed - True
+
+
+### Example
```

Comment:
This should probably be heading 2 (i.e. ##) since its not a sub-heading of Source Node (the section above it)